tests/libvmi/network.go

```diff
@@ -79,12 +79,12 @@ func InterfaceWithMac(iface *kvirtv1.Interface, macAddress string) *kvirtv1.Inte
 }
 
 // MultusNetwork returns a Network with the given name
-func MultusNetwork(networkName string) *kvirtv1.Network {
+func MultusNetwork(nadName, networkName string) *kvirtv1.Network {
```

Comment:
nit: I think the order of the arguments need to be per in reverse.
The subject here is the network, so its name should be first and only then details of its content.

Also, before this change, we assumed both will be called the same, but now it is not the case so you need to rename the existing argument to be less confusing. So how about changing it to just `name`?
The doc comment also needs to be updated.

Reply:
done